vsphere-mixin/panels.libsonnet

```diff
@@ -0,0 +1,1472 @@
+          'Top memory usage by resource pools',
+          targets=[t.topMemoryUsageResourcePools],
+          description='The resource pools with the highest memory usage in the datacenter.'
+        )
+        + g.panel.timeSeries.options.legend.withDisplayMode('table')
+        + g.panel.timeSeries.standardOptions.withUnit('mbytes'),
+
+      topCPUShareResourcePools:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Top CPU shares by resource pools',
+          targets=[t.topCPUShareResourcePools],
+          description='The resource pools with the highest amount of CPU shares allocated in the datacenter.'
+        )
+        + g.panel.timeSeries.options.legend.withDisplayMode('table')
+        + g.panel.timeSeries.standardOptions.withUnit('shares'),
+
+      topMemoryShareResourcePools:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Top memory shares by resource pools',
+          targets=[t.topMemoryShareResourcePools],
+          description='The resource pools with the highest amount of memory shares allocated in the datacenter.'
+        )
+        + g.panel.timeSeries.options.legend.withDisplayMode('table')
+        + g.panel.timeSeries.standardOptions.withUnit('shares'),
+
+      topCPUUtilizationHosts:
+        commonlib.panels.memory.timeSeries.usagePercent.new(
+          'Top CPU utilization by ESXi hosts',
+          targets=[t.topCPUUtilizationHosts],
+          description='The ESXi hosts with the highest CPU utilization in the datacenter.'
+        )
+        + g.panel.timeSeries.options.legend.withDisplayMode('table')
+        + g.panel.timeSeries.standardOptions.withUnit('percent'),
+
+      topMemoryUtilizationHosts:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Top memory usage by ESXi hosts',
+          targets=[t.topMemoryUtilizationHosts],
+          description='The ESXi hosts with the highest memory utilization in the datacenter.'
+        )
+        + g.panel.timeSeries.options.legend.withDisplayMode('table')
+        + g.panel.timeSeries.standardOptions.withUnit('percent'),
+
+      topDiskAvgLatencyHosts:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Top avg disk latency by ESXi hosts',
+          targets=[t.topDiskAvgLatencyHosts],
+          description='The ESXi hosts with the highest average disk latency in the datacenter.'
+        )
+        + g.panel.timeSeries.options.legend.withDisplayMode('table')
+        + g.panel.timeSeries.standardOptions.withUnit('ms'),
+
+      topPacketErrorRateHosts:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Top packet errors by ESXi hosts',
+          targets=[t.topPacketErrorRateHosts],
+          description='The ESXi hosts with the highest percentage of packet errors in the datacenter.'
+        )
+        + g.panel.timeSeries.options.legend.withDisplayMode('table')
+        + g.panel.timeSeries.standardOptions.withUnit('percent'),
```

Comment:
Same issue for these four.

Any one of these options would suffice:
- List based
- Table (current) with more height
- Table (current) right-aligned